libs/game/info.ts

```diff
@@ -711,12 +712,12 @@ namespace info {
             const oldScore = state.score || 0;
             state.score = (value | 0);
 
-            if (state.scoreReachedHandler && (
-                (oldScore < state.scoreReachedHandler.score && state.score >= state.scoreReachedHandler.score) ||
-                (oldScore > state.scoreReachedHandler.score && state.score <= state.scoreReachedHandler.score)
-            )) {
-                state.scoreReachedHandler.handler();
-            }
+            state?.scoreReachedHandler.forEach(srh => {
```

Comment:
Static typescript code (~~user level code) is compiled using [typescript 2.6](https://github.com/microsoft/pxt/tree/master/pxtcompiler/ext-typescript), so can't use optional chaining unfortunately as that wasn't added tille like 3.7 or so